en/firefox/privacy-hub.ftl

```diff
@@ -0,0 +1,103 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/privacy/
+
+# HTML page title
+firefox-privacy-hub-firefox-privacy-promise = { -brand-name-firefox } Privacy Promise
+
+# HTML page description
+firefox-privacy-hub-firefox-takes-less-data-keeps = { -brand-name-firefox } takes less data, keeps it safe, and with no secrets.
+
+# The strong tag wraps a word that gets highlighted with a colorful underline for emphasis. The underline breaks if it is on two words, please omit the strong tags if they need to be around multiple words in your language
+firefox-privacy-hub-firefox-products-are-designed = { -brand-name-firefox } products are designed to protect your <strong>privacy</strong>
+
+# Sub navigation title
+firefox-privacy-privacy = Privacy
+
+# Sub navigation link
+firefox-privacy-our-promise = Our Promise
+
+# Sub navigation link
+firefox-privacy-our-products = Our Products
+
+firefox-privacy-hub-you-should-be-able-to-decide = You should be able to decide who sees your personal info. Not just among your friends, but with every advertiser and company on the internet — including us.
+firefox-privacy-hub-thats-why-everything-we-make = That’s why everything we make and do honors our Personal Data Promise
+firefox-privacy-hub-take-less = Take Less
+firefox-privacy-hub-we-make-a-point-of-knowing = We make a point of knowing less about you
+firefox-privacy-hub-all-tech-companies-collect = All tech companies collect data to improve their products. But it doesn’t need to include so much of your personal info. The only data we want is the data that serves you in the end. We ask ourselves: do we actually need this? What do we need it for? And when can we delete it?
+firefox-privacy-hub-keep-it-safe = Keep it safe
+firefox-privacy-hub-we-do-the-hard-work-to-protect = We do the hard work to protect your personal info
+firefox-privacy-hub-data-security-is-complicated = Data security is complicated — or at least it should be. Which is why we take the extra steps to classify the data we have, maintain rules for how we store and protect each type, and never stop iterating on our processes. We prioritize your privacy. We invest in it. We’re committed to it. We even teach other companies how to do it.
```

Comment:
We only include a link here for English (using a hard-coded string), since the link destination is also in English only. Other locales get the same sentance but without the link, hence why it's omitted here in the translated string.

https://github.com/mozilla/bedrock/blob/master/bedrock/firefox/templates/firefox/privacy/index.html#L77-L90